core/db/migrate/20250526114936_create_spree_imports.rb

```
@@ -0,0 +1,15 @@
+class CreateSpreeImports < ActiveRecord::Migration[7.2]
+  def change
+    create_table :spree_imports do |t|
+      t.references :user, null: true, foreign_key: { to_table: 'spree_users' }, index: true
+      t.references :store, null: false, foreign_key: { to_table: 'spree_stores' }, index: true
+      t.string :type, null: :false
```

RuboCop (1.75.5)

[warning] Symbol with a boolean name - you probably meant to use `false`.

(Lint/BooleanSymbol)